src/bundle/Controller/ContentViewController.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/**
+ * File containing the ContentViewController class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace EzSystems\HybridPlatformUiBundle\Controller;
+
+use eZ\Publish\API\Repository\Values\Content\Location;
+use eZ\Publish\API\Repository\Values\Content\VersionInfo;
+use eZ\Publish\Core\MVC\Symfony\Controller\Controller;
+use eZ\Publish\Core\MVC\Symfony\View\ContentView;
+
+class ContentViewController extends Controller
+{
+    protected $defaultSortFields = [
+        Location::SORT_FIELD_NAME => 'sort.name',
+        Location::SORT_FIELD_PRIORITY => 'sort.priority',
+        Location::SORT_FIELD_MODIFIED => 'sort.modified',
+        Location::SORT_FIELD_PUBLISHED => 'sort.published'
+    ];
+
+    protected $otherSortFields = [
+        Location::SORT_FIELD_PATH => 'sort.path',
+        Location::SORT_FIELD_CLASS_IDENTIFIER => 'sort.content.type.identifier',
+        Location::SORT_FIELD_SECTION => 'sort.section',
+        Location::SORT_FIELD_DEPTH => 'sort.depth',
+        Location::SORT_FIELD_CLASS_NAME => 'sort.content.type.name',
+    ];
+
+    public function detailsTabAction(ContentView $view)
+    {
+        $versionInfo = $view->getContent()->getVersionInfo();
+        $contentInfo = $versionInfo->getContentInfo();
+
+        $sectionService = $this->getRepository()->getSectionService();
+        $section = $sectionService->loadSection($contentInfo->sectionId);
+
+        $userService = $this->getRepository()->getUserService();
+        $creator = $userService->loadUser($contentInfo->ownerId);
```

Comment:
+1 for using `Repository:sudo()`.
We want to display the first name and the last name of the user account even if the current logged in user does not have the right (content/read).